src/rules/fenced-code-language.js

```diff
@@ -90,7 +99,16 @@ export default {
 
 				if (required.size && !required.has(node.lang)) {
 					context.report({
-						loc: node.position,
+						loc: {
+							start: node.position.start,
+							end: {
+								line: node.position.start.line,
+								column:
+									sourceCode.lines[
+										node.position.start.line - 1
+									].length + 1,
```

Comment:
[Bug] Any meta text attached to the code block will also be captured by this. But meta text isn't what's relevant.

````md
```js foo
````

````md
```js filename="src/index.js"
````

Proposal: how about limiting the range to just the <code>\`\`\`</code>, as well as the language if disallowed?